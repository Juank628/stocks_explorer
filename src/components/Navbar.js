import React from 'react';
import { Link } from 'react-router-dom';
import backIcon from '../assets/back_icon.png';
import settingsIcon from '../assets/settings_icon.png';
import styles from './Navbar.module.scss';

export default function Navbar() {
  return (
    <header className={styles.mainContainer}>
      <Link to="/"><img className={styles.back} src={backIcon} alt="back" /></Link>
      <h1 className={styles.title}>Sector performance</h1>
      <Link to="/"><img className={styles.settings} src={settingsIcon} alt="back" /></Link>
    </header>
  );
}
