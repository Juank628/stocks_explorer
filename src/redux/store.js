import { configureStore } from '@reduxjs/toolkit';
import stocksReducer from './slices/stocks';
import detailsReducer from './slices/details';

const store = configureStore(
  {
    reducer: { stocksReducer, detailsReducer },
  },
);

export default store;
