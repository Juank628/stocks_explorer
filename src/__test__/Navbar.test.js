import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../redux/store';
import '@testing-library/jest-dom';
import Navbar from '../components/Navbar';

describe('Navbar', () => {
  test('renders navbar', () => {
    const element = render(
      <Provider store={store}>
        <BrowserRouter>
          <Navbar />
        </BrowserRouter>
      </Provider>,
    );
    expect(element.toJSON).toMatchSnapshot();
  });
});
