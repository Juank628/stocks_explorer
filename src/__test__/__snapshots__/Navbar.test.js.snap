// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders navbar 1`] = `undefined`;
